server/models/video/thumbnail.ts

```diff
@@ -72,6 +73,18 @@ export class ThumbnailModel extends Model<ThumbnailModel> {
   })
   VideoPlaylist: VideoPlaylistModel
 
+  @ForeignKey(() => TimecodeThumbnailManifestModel)
+  @Column
+  manifestId: number
```

Comment:
`timecodeThumbnailManifestId`

I know it's long but I don't want to rename a foreign key in the future ^^